cookbook/tools/browserbase_tools.py

```diff
@@ -0,0 +1,34 @@
+from os import getenv
+
+from agno.agent import Agent
+from agno.tools.browserbase import BrowserbaseTools
+
```

Comment:
Add some instructions here on the env variables to set

Reply:
got it